varsim_validator.py

```diff
@@ -0,0 +1,117 @@
+import glob
+import tempfile
+import re
+from distutils.version import LooseVersion
+from liftover_restricted_vcf_map import lift_vcfs, lift_maps
+from generate_small_test_ref import gen_restricted_ref_and_vcfs 
+from varsim import varsim_main, get_version, check_java, get_loglevel, makedirs, RandVCFOptions, RandDGVOptions
+import json
+
+MY_DIR = os.path.dirname(os.path.realpath(__file__))
+VARSIMJAR = os.path.realpath(os.path.join(MY_DIR, "VarSim.jar"))
+
+def aggregate_reports(sample_reports, samples, variant_type="all"):
+    if variant_type == "all":
+        summary_report = sample_reports[samples[0]]["num_true_correct"]["all_data"]["sum_count"]
+    elif variant_type == "snp":
+        summary_report = sample_reports[samples[0]]["num_true_correct"]["data"]["SNP"]["sum_count"]
+    elif variant_type == "ins":
+        summary_report = sample_reports[samples[0]]["num_true_correct"]["data"]["Insertion"]["sum_count"]
+    elif variant_type == "del":
+        summary_report = sample_reports[samples[0]]["num_true_correct"]["data"]["Deletion"]["sum_count"]
+        
+    keys_for_aggregation = ["_TP", "_FP", "_TN", "_FN", "_T", "fp", "tn", "fn", "tp", "t"]
+    for sample in samples[1:]:
+        if variant_type == "all":
+            sample_report = sample_reports[sample]["num_true_correct"]["all_data"]["sum_count"]
+        elif variant_type == "snp":
+            sample_report = sample_reports[sample]["num_true_correct"]["data"]["SNP"]["sum_count"]
+        elif variant_type == "ins":
+            sample_report = sample_reports[sample]["num_true_correct"]["data"]["Insertion"]["sum_count"]
+        elif variant_type == "del":
+            sample_report = sample_reports[sample]["num_true_correct"]["data"]["Deletion"]["sum_count"]
+
+        for key in keys_for_aggregation:
+            summary_report[key] += sample_report[key]
+
+    summary_report["fdr"] = float(summary_report["fp"]) / float(summary_report["fp"] + summary_report["tp"]) * 100
+    summary_report["tpr"] = float(summary_report["tp"]) / float(summary_report["t"]) * 100
+    summary_report["ppv"] = 100.0 - summary_report["fdr"]
+    summary_report["f1"] = 2.0 * float(summary_report["ppv"]  * summary_report["tpr"]) / float(summary_report["ppv"] + summary_report["tpr"])
+
+    return summary_report
+    
+
+def varsim_multi_validation(reference, regions, samples, varsim_dir, variants_dir, out_dir, vcfcompare_options=""):
+    logger = logging.getLogger(varsim_multi_validation.__name__)
+
+    bed_options = "-bed {}".format(regions) if regions else ""
+    vcfcompare_options += " " + bed_options
+
+    sample_reports = {}
+    for sample in samples:
+        sample_dir = os.path.join(out_dir, sample)
+        makedirs([sample_dir])
+
+        sample_truth = os.path.join(varsim_dir, sample, "out", "lifted", "truth.vcf")
+        sample_called = os.path.join(variants_dir, sample, "{}.vcf".format(sample))
+        sample_called = os.path.join(variants_dir, sample, "fb_0.35.vcf")
```

Comment:
Hi Marghoob, this line can be removed now because I have changed vcf to `sample.vcf`, I just tested it on the 10 sample example and it works well (without this line).